background/logging.js

```diff
@@ -0,0 +1,32 @@
+require('app-module-path').addPath(__dirname + '../../app');
+
+var path = require('path');
+
+var nconf = require('nconf');
+var bunyan = require('bunyan');
+
+formatBunyanInfoLog = function() {};
+formatBunyanInfoLog.prototype.write = function (rec) {
+    console.log('[%s] %s: %s',
```

Comment:
Would it be possible to have this as an option so people can change the logging style if they wanted to? We should also probably document this in a wiki or something so if anyone wants to use an external logger such as graylog or ELK they can easily import it.